labgrid/pytestplugin/fixtures.py

```diff
@@ -1,8 +1,12 @@
 import os
 import subprocess
+from typing import Iterator
```

Comment:
```suggestion
from collections.abc import Iterator
```

`collections.abc` was used in `labgrid/environment.py`, so would be nice to be consistent.